.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
```

Comment:
[OPTIONAL] note: I have only made this optional because you have done an incredible job on your project. However, this is part of the project requirements and I highly suggest that you at least try to implement it and make sure that you follow these guidelines in the upcoming projects. This will help follow the best practices when writing JavaScript and write better scripts in your software development career.

Since you are using JavaScript for this milestone, you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.

Kindly update your `linters.yml` file and replace it with the one from the [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) Linters Config repository.

![image](https://user-images.githubusercontent.com/38088818/167582683-57ee2c36-0b50-4a9c-9042-a74e97e9e88b.png)

- Please make sure your ESlint linter checks are passing after updating the config files.

Your checks should have the ESlint checks as well:
![Eslintchecks](https://user-images.githubusercontent.com/38088818/168270144-5cf68371-6354-4476-90fa-94debe7cd848.png)